src/planner/dart/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+if(CMAKE_COMPILER_IS_GNUCXX)
+  if(OMPL_VERSION VERSION_GREATER 1.2.0 OR OMPL_VERSION VERSION_EQUAL 1.2.0)
+    if(Boost_VERSION VERSION_LESS 106501)
```

Comment:
We do not need to check boost version here since this subdirectory does not have ompl code.

Reply:
This is related to the below comment as well: this PR is set up for https://github.com/personalrobotics/aikido/pull/566. Sorry, I totally should have made that more clear: because the ultimate goal here is deprecating a lot of the old `robot/util` stuff, we need to move some of the `planTo` methods that use ompl into this new `dart_planner` module. That's what https://github.com/personalrobotics/aikido/pull/566 starts to do, but we set up the ompl dependency here 😄